pkg/webui/console/store/selectors/events.js

```diff
@@ -0,0 +1,33 @@
+// Copyright © 2019 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const storeSelector = (state, props) => state[props.id]
```

Comment:
It really took me a while to figure out what these are. `selectors` clashes with our term for `selectors` in the JS SDK. I would really prefer the name `mappers`, which is more in line with their usage in `mapStateToProps()`.

Reply:
no, a selector is a well known term in the redux world, not a mapper.